### DISCUSSION TITLE
Backing Up and Migrating Containers with Volumes to new Host

### DISCUSSION THREAD

**Discussion by OGGary** [netbox-community/netbox-docker#443] [Getting Help] [answered]
Hello devs and network engineers I have a problem and I hope someone will be there to help. I want to copy my postgresql container with volume. I want to migrate the data of my netbox to another system. Is there anyway to keep the data I had in my netbox? thx

**Comment by cimnine** [comment, marked as answer]
See https://github.com/netbox-community/netbox-docker/wiki/Troubleshooting#database-operations

Also don't forget to migrate the data files, e.g. picture that users have uploaded. They are in a Docker volume that is attached to the Netbox container.

**Reply by OGGary** [reply]
Thanks I will try that out.

**Reply by OGGary** [reply]
Thank you man!
